src/screens/FeedScreen/hooks/useFeed.js

```diff
@@ -103,7 +103,7 @@ const useFeed = () => {
       return COLORS.redalert;
     }
 
-    return COLORS.lightgrey;
+    return COLORS.gray100;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that this change aligns with the overall theme and design of your application. If `COLORS.gray100` is not defined in your `COLORS` object, this will result in an error.
```diff
-    return COLORS.lightgrey;
+    return COLORS.gray100;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->